src/Service/EdroneService.php

```diff
@@ -57,11 +83,11 @@ public function subscribe(?string $firstName, string $email): void
     }
 
     public function createProductCategoryStruct(
-        array $navigationTree,
+        array                     $navigationTree,
         SalesChannelProductEntity $product
-    ): ?EdroneProductCategoryStruct {
+    ): ?EdroneProductCategoryStruct
+    {
         $this->searchBreadcrumbs($navigationTree, $product);
```

Comment:
Wiem, że to było tutaj wcześniej ale przy okazji chyba warto to naprawić... Nie kumam czemu $this->breadcrumb jest polem tej klasy a nie zwykłą zmienną?